s4e-backend/src/test/java/pl/cyfronet/s4e/TestDbHelper.java

```diff
@@ -15,6 +15,7 @@
     private final WMSOverlayRepository wmsOverlayRepository;
     private final PRGOverlayRepository prgOverlayRepository;
     private final SldStyleRepository sldStyleRepository;
+    private final InvitationRepository invitationRepository;
```

Comment:
Since you don't need to call it in the method `::clean` I'd remove it from the dependencies too.